server/routes/userRoutes.js

```diff
@@ -60,13 +60,13 @@ module.exports = (db) => {
         if (!user) {
           return res.status(400).send("There was an issue registering.");
         }
-        console.log("successful registration");
         req.session.userId = user.id;
-        res.status(200).send("User created!");
-      })
-      .then((user) => {
-        dbHelper.getUserWithID(req.session.userId).then((user) => {});
+        // console.log("successful user set");
+        dbHelper.addQuotaForUser(user.id, "3 hours");
```

Comment:
This needs a `.then` that runs line 67, and a `.catch` returning status 500 and error.

Remember to do `res.json` not `.send`.

Reply:
Done

Reply:
You misunderstood, I meant you should do 
```
dbHelper.addQuotaForUser(user.id, "1.5 hours")
.then(()=>res.status(200).json("User created!"))
.catch((err) => res.status(500).json("Invalid Request", err));
```

Every dbHelper call should have a .then and .catch. You can't do something like
```
dbHelper.addQuotaForUser(user.id, "3 hours");
return res.status(200).json("User created!");
```

Reply:
Please review last commit